README.md

```diff
@@ -110,6 +110,12 @@ simple_things_entity_audit:
     entity_manager: custom
 ```
 
+If there is no need to create foreign keys for any reason, you can specify the corresponding parameter:
```

Comment:
```suggestion
If you need to explicitly discard the foreign keys inferred from the audited entities, you can use the `disable_foreign_keys` parameter:
```
BTW, providing context for the use cases like the example mentioned in the PR's description can be useful here.